ui/src/components/Nodes/NodeActionsDropdown.vue

```diff
@@ -0,0 +1,116 @@
+    <FeatherDropdownItem
+      v-for="linkItem in linkItems"
+      :key="linkItem.name"
+      @click="onNodeLink(linkItem.name, node)">
+      <span class="node-menu-item">{{ linkItem.label }}</span>
+    </FeatherDropdownItem>
+  </FeatherDropdown>
+</template>
+
+<script setup lang="ts">
+import { FeatherButton } from '@featherds/button'
+import { FeatherDropdown, FeatherDropdownItem } from '@featherds/dropdown'
+import { FeatherIcon } from '@featherds/icon'
+import MoreVert from '@featherds/icon/navigation/MoreVert'
+import { markRaw, PropType } from 'vue'
+import { Node } from '@/types'
+
+const props = defineProps({
+  baseHref: {
+    required: true,
+    type: String
+  },
+  node: {
+    required: true,
+    type: Object as PropType<Node>
+  },
+  triggerNodeInfo: {
+    required: true,
+    type: Function as PropType<(node: Node) => void>
+  }
+})
+
+const menu = markRaw(MoreVert)
+
+const linkItems = [
+  { name: 'events', label: 'Events' },
+  { name: 'alarms', label: 'Alarms' },
+  { name: 'view-outages', label: 'Outages' },
+  { name: 'assets', label: 'Assets' },
+  { name: 'metadata', label: 'Metadata' },
+  { name: 'hardware', label: 'Hardware Info' },
+  { name: 'availability', label: 'Availability' },
+  { name: 'graphs', label: 'Resource Graphs' },
+  { name: 'rescan', label: 'Rescan' },
+  { name: 'admin', label: 'Admin' },
+  { name: 'updateSnmp', label: 'Update SNMP' },
+  { name: 'schedule-outage', label: 'Schedule an Outage' },
+  { name: 'topology', label: 'View in Topology' }
+]
+
+const onNodeLink = (name: string, node: Node) => {
+  const link = mapLink(name, node)
+  window.location.assign(`${props.baseHref}${link}`)
+}
+
+const mapLink = (name: string, node: Node) => {
+  switch (name) {
+    case 'events':
+      return `event/list?filter=node%3D${node.id}`
+    case 'alarms':
+      return `alarm/list.htm?filter=node%3D${node.id}`
+    case 'view-outages':
+      return `outage/list.htm?filter=node%3D${node.id}`
+    case 'assets':
+      return `asset/modify.jsp?node=${node.id}`
+    case 'metadata':
+      return `element/node-metadata.jsp?node=${node.id}`
+    case 'hardware':
+      return `hardware/list.jsp?node=${node.id}`
+    case 'availability':
+      return `element/availability.jsp?node=${node.id}`
+    case 'graphs':
+      return `graph/chooseresource.jsp?node=${node.id}&reports=all`
+    case 'rescan':
+      return `element/rescan.jsp?node=${node.id}`
+    case 'admin':
+      return `admin/nodemanagement/index.jsp?node=${node.id}`
+    case 'updateSnmp':
+      // TODO: Get IP Address
+      return `admin/updateSnmp.jsp?node=${node.id}&ipaddr=0.0.0.0`
+    case 'schedule-outage':
+      return `admin/sched-outages/editoutage.jsp?newName=${node.label}&addNew=true&nodeID=${node.id}`
+    case 'topology':
+      return `topology?provider=Enhanced+Linkd&szl=1&focus-vertices=${node.id}`
+    default: return ''
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.focus-icon {
+  cursor: pointer;
+}
```

Comment:
there's a few of these classes with just `cursor: pointer` in this PR, which is fine, but I think there is still the global class `pointer` which does the same thing and could possibly replace these.

also, is this class used? doesn't appear in the template

Reply:
I'll try the global `pointer` class

Reply:
Used global `pointer`, removed unused class.